Core/include/Acts/TrackFinding/TrackSelector.hpp

```diff
@@ -455,23 +454,17 @@ inline TrackSelector::TrackSelector(
   m_isUnbinned = false;
   if (m_cfg.nEtaBins() == 1) {
     static const std::vector<double> infVec = {0, inf};
```

Comment:
this could be defined in `EtaBinnedConfig` and reused here

Reply:
Is this one used elsewhere? I think if it is only needed here, it is better to keep local. But maybe better to make it `static constexpr`.

Reply:
The same value is used in two other places
- https://github.com/acts-project/acts/blob/f6877db335b0fe340ac5d622d274c49f318f19f3/Core/include/Acts/TrackFinding/TrackSelector.hpp#L151-L153
- https://github.com/acts-project/acts/blob/f6877db335b0fe340ac5d622d274c49f318f19f3/Core/include/Acts/TrackFinding/TrackSelector.hpp#L169-L172

Not too bad but if we repeat it it might be worth having a `static const` or `static constexpr` if that is possible. Additionally we could initialize `absEtaEdges` to that value and remote it from the constructors if we use the default.

Reply:
OK, I added `absEtaEdgesDefault`. Unfortunately, it can't be `constexpr` because it is a vector used outside `constexpr` context.

Reply:
Alternatively we can use a static function with returns this kind of vector by value or by reference with static lifetime. Not great but it would make clear what `{0, inf}` means and why we check against it.

Alternatively `std::array` and then construct the vector from it by `(begin, end)` but also not pretty.

That's all I have - I don't really have a preference since it is not as easy as I hoped it to be

Reply:
I reverted the change, but used the default setting for the object variable.